_stdlib/component_defines.dm

```diff
@@ -27,6 +27,13 @@
 /// each component of the same type is consulted as to whether the duplicate should be allowed
 #define COMPONENT_DUPE_SELECTIVE		5
 
+// "so if a proc bound to a signal returns a value it's treated as bit flags" - Pali
+/// Success
+#define COMSIG_RETURN_SUCCESS 1
+/// Failure
+#define COMSIG_RETURN_FAILURE 2
+/// return early (don't complete rest of proc)
+#define COMSIG_RETURN_EARLY 4
```

Comment:
I think these make sense to have, but if they don't, let know.

Reply:
They do but it might be worth it to maybe use higher powers of two so specific signals can use these smaller ones without having to worry about collisions. Or maybe that's a bad idea and instead there should be a define that says at which bit signal-specific bitflags can start.
Also I'm not too sure about the `COMSIG_` prefix here since that seems to be more or less reserved for actual signals.